ecs/backend.go

```diff
@@ -32,6 +29,9 @@ import (
 	"github.com/docker/compose-cli/context/cloud"
 	"github.com/docker/compose-cli/context/store"
 	"github.com/docker/compose-cli/errdefs"
+
```

Comment:
goimports, this should be above our local imports

Reply:
I like it so much spending energy with such stupid things :P

Reply:
😬

I'm very nitpicky about code style and my editor does this magically for me.

Reply:
Goland also manage imports but from time to time new imports don't get introduce in the right place. I wish `goimports` had a mode to _ignore_ sections and just order things in canonical way os we don't have to worry about this.